### PR TITLE
Release notes for Foreman 2.2.2

### PR DESCRIPTION
**Pull request by upadhyeammit**

### CODE REVIEW

File: _includes/manuals/2.2/1.2_release_notes.md [hunk 1]
```diff
@@ -26,8 +26,25 @@ Work has been started on creating a new host detail page. The initial implementa
 
 * VNC console connection may fail to connect due to browser security settings. This will be fixed in 2.2.2 and 2.3.0. The patch fixing it can be safely applied manually if needed. ([#31317](https://projects.theforeman.org/issues/31317))
```
Comment (tbrisker) [inline _includes/manuals/2.2/1.2_release_notes.md]: This is fixed now, we can drop the known issues section

**Review by tbrisker — APPROVED** [review]

**Review by tbrisker — CHANGES_REQUESTED** [review]
I think you missed something in the last push, the release notes are missing now

**Comment by upadhyeammit** [comment]
> I think you missed something in the last push, the release notes are missing now

pushed the corrected changes, thank you !

**Review by tbrisker — APPROVED** [review]
Thanks @upadhyeammit !

**Pull request merged**